code/modules/vehicles/janicart.dm

```diff
@@ -34,30 +34,30 @@
 		return
 	..()
 
-/obj/vehicle/train/cargo/engine/pussywagon/turn_on()
+/obj/vehicle/train/cargo/engine/janicart/turn_on()
 	if(!key)
 		return
 	else
 		..()
 		update_stats()
 
-		verbs += /obj/vehicle/train/cargo/engine/pussywagon/verb/toggle_mop
-		verbs += /obj/vehicle/train/cargo/engine/pussywagon/verb/toggle_hoover
+		verbs += /obj/vehicle/train/cargo/engine/janicart/verb/toggle_mop
+		verbs += /obj/vehicle/train/cargo/engine/janicart/verb/toggle_hoover
 
-/obj/vehicle/train/cargo/engine/pussywagon/turn_off()
+/obj/vehicle/train/cargo/engine/janicart/turn_off()
 	..()
 
 	if(tow)
-		if(istype(tow,/obj/vehicle/train/cargo/trolley/pussywagon))
-			var/obj/vehicle/train/cargo/trolley/pussywagon/PW = tow
+		if(istype(tow,/obj/vehicle/train/cargo/trolley/janicart))
+			var/obj/vehicle/train/cargo/trolley/janicart/PW = tow
 			if(PW.mopping)
 				PW.mop_toggle()
 			if(PW.hoover)
```

Comment:
REPLACE WITH SOCIETY-FRIENDLY INITIALS
gosh im SHAKING RIGHT NOW
HOW DARE YOU NOT REPLACE THAT?
Do you even care?